vaksina/fhir_parser.py

```diff
@@ -185,20 +185,23 @@ def parse_bundle_to_persons(self, bundle):
                     # FIXME: debug logger
                     continue
 
-                immunizations.append(self.parse_immunization_record(resource))
+                refid = resource["patient"]["reference"]
+                immunizations = self.parse_immunization_records(resource)
```

Comment:
Just realized I am basically assuming that #20 was merged. If that isn't true, then this line should be changed to `immunizations = [self.parse_immunization_record(resource)]`